docs/runtime.md

```diff
@@ -0,0 +1,138 @@
+# RedisGears Functions Runtime
+
+Python RedisGears functions are run using an embedded Python interpreter. Each function uses a separate sub-interpreter. All functions share the same environment and dependencies. The environment is imported with several defaults.
```

Comment:
The subinterpreter was currently removed on 0.9.0 and isolation is achieved using different global dictionaries (which seems good enough), we might decide to return the subinterpreters in the future ..

Reply:
Fixed via 079f280